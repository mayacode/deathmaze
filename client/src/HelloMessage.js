import React from 'react';
import { string } from 'prop-types';

export function HelloMessage({ name }) {
  return <div>Hello {name}</div>;
}

HelloMessage.propTypes = {
  name: string,
};

export default HelloMessage;
