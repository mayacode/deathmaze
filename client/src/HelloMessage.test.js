import React from 'react';
import { shallow } from 'enzyme';

import HelloMessage from './HelloMessage';

describe('<HelloMessage />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HelloMessage name="Ala" />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render name Ala', () => {
    expect(wrapper.text()).toBe('Hello Ala');
  });
});
