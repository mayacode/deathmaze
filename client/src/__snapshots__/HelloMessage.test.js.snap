// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloMessage /> should match snapshot 1`] = `
<div>
  Hello 
  Ala
</div>
`;
